client/src/Components/Pages/ViewLocation.js

```diff
@@ -0,0 +1,118 @@
+import { useEffect, useState, useMemo } from 'react'
+import React from 'react';
+import { GoogleMap, MarkerF, InfoWindow, useJsApiLoader } from '@react-google-maps/api';
+import './ViewLocation.css';
+import usePlacesAutoComplete, {getGeocode, getLatLng } from 'use-places-autocomplete';
+
+
+export default function ViewLocation() {
+  const [location, setLocation] = useState({ lat: 35.6761919, lng: 139.6503106 });
+  const libraries = useMemo(() => ['places'], []);
+
+  useEffect(() => {
+    async function displayLocation(address) {
+      try {
+        const results = await getGeocode({ address });
+        if (!results.ok) {
+          throw new Error(`error ${results.status}`);
+        }
+        const { lat, lng } = await getLatLng(results[0]);
+        setLocation({ lat, lng});
+      } catch(err) {
+        console.error('Error!:', err);
+      }
+    }
+  }, []);
+
+    const containerStyle = {
+      width: '70vw',
+      height: '70vh'
+    };
+
+    const { isLoaded } = useJsApiLoader({
+      id: 'google-map-script',
+      googleMapsApiKey: "AIzaSyAA9yzP9GJ7dLzvbAD5ybpzCm1ImSy4TqA",
+      libraries
+    })
+
+    const [map, setMap] = React.useState(null)
+
+    const onLoad = React.useCallback(function callback(map) {
```

Comment:
No arrow function?

I can tell you copy/pasted this code. Clean it up to fit with your style!